src/backend/distributed/utils/citus_ruleutils.c

```diff
@@ -676,6 +676,26 @@ pg_get_tablecolumnoptionsdef_string(Oid tableRelationId)
 }
 
 
+/*
+ * pg_get_tableowner_string
+ */
+char *
+pg_get_tableowner_string(Oid relationId)
```

Comment:
Minor note: There is `TableOwnerResetCommand()` implemented for Citus MX. Can we use it here, or is there a reason for not doing that?